contracts/DEX/SimpleDex.sol

```diff
@@ -71,4 +88,61 @@ contract SimpleDex {
         uint256 amountOutstanding = order.amount.sub(order.filled);
         return (order, fillAmount, amountOutstanding, order.maker);
     }
-}
+
+    /**
+    * @notice Matches two orders that have already both been made. Has the same
+    *         validation as takeOrder 
+    */
+    function _matchOrder(
+        uint256 order1Id,
+        uint256 order2Id
+    ) internal returns (uint256) {
```

Comment:
So is this function essentially making it so two orders created from `makeOrder`, can both be filled against eachother?